src/Context/Extensions/ProfilingStatus.cs

```diff
@@ -0,0 +1,25 @@
+﻿namespace MongoDB.Extensions.Context
+{
+    public class ProfilingStatus
```

Comment:
if this is NET 5.0, a Record could do this and avoid some code oh, and init-only props ;)

Reply:
Hahaha... yeah its a good idea for the future... but at the moment I would like to have backward compatibility to dotnet core 3.1